lib/ui/src/components/sidebar/Tree.stories.tsx

```diff
@@ -95,3 +96,21 @@ export const SingleStoryComponents = () => {
     />
   );
 };
+
+// node must be selected, highlighted, and focused
+// in order to tab to 'Skip to canvas' link
+export const SkipToCanvasLinkFocused = {
+  args: {
+    isBrowsing: true,
+    isMain: true,
+    refId,
+    data: stories,
+    highlightedRef: { current: { itemId: 'tooltip-tooltipbuildlist--default', refId } },
+    setHighlightedItemId: log,
+    selectedStoryId: 'tooltip-tooltipbuildlist--default',
+    onSelectStoryId: () => {},
+  },
+  play: () => {
+    screen.getByTestId('node--skip').focus();
```

Comment:
@darleendenno try returning the promise so Chromatic can `await` it properly

Reply:
Feels weird to me -- but I can give it a whirl

![image](https://user-images.githubusercontent.com/17681528/128033131-c41d3cf2-dfab-42de-919f-b90ec941e641.png)